static/sass/_patterns_takeunders.scss

```diff
@@ -0,0 +1,23 @@
+/// homepage takeunders
+@mixin maas-p-takeunders {
+  .p-takeunder {
+    border-radius: .125rem;
+    color: $color-x-light;
```

Comment:
Settings the color without linking it with a background makes this pattern fragile. Please use the `is-dark`, `is-light` pattern used in the `p-strip--image` pattern.